### PR TITLE
Update v5e perf results and fix rto_setup for GCE

### PR DESCRIPTION
**Pull request by raymondzouu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -232,15 +232,24 @@ For a 52B model. See full run configs in `MaxText/configs/` as `1xv4-384.sh` and
 
 For a 16B model. See full run configs in `MaxText/configs/` as `16b.sh`.
```
Comment (rwitten) [inline README.md:233]: can you add the 64B and 128B results as well?

Comment (rwitten) [reply README.md:233]: Generally, we want a sensible getting started page for users that show our full results. I think in the main README we should just say we get between 55% MFU and 67% MFU between 256 and 4096 v5e chips and 16B to 128B params. Then we should link to the v5e results MD page.

**Review by rwitten — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raymondzouu) [reply README.md]: Done. Added the 64B and 128B results and updated the v5e results md page.

File: v5e_high_performance.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+6. Upload the docker image to your GCP project.
+     ```
+     # Be sure to have your project and compute/zone set
+     gcloud config set project ${PROJECT}
+     gcloud config set compute/zone ${ZONE}
+
+     # Upload docker image
+     bash docker_upload_runner.sh CLOUD_IMAGE_NAME=${YOUR_IMAGE_NAME}
+     ```
+
```
Comment (Obliviour) [inline v5e_high_performance.md:130]: Not sure if you want to add a step  (6.5) about  `xpk cluster cacheimage` to enable faster start times.

Comment (Obliviour) [reply v5e_high_performance.md:130]: https://github.com/google/maxtext/blob/main/xpk/README.md#cluster-cacheimage
and https://github.com/google/maxtext/blob/main/xpk/xpk-large-scale-guide.sh#L482

**Review by Obliviour — APPROVED** [review]

**Review by rwitten — APPROVED** [review]

File: v5e_high_performance.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+## Reproduction Instructions
+Depending on your organization's set up, these instructions may vary. Here we provide instructions for running on a vanilla GCE setup or a GKE with XPK setup. Feel free to reach out with questions if your organization has a unique setup.
+
+### Running on GCE
+1. Create a custom MTU network to optimize network performance and give it firewall rules. If you are unable to complete this step, you may skip it. This step is not necessary, however performance degradation from scaling up the number of v5e pods over DCN will become more noticable.
+     
+     Create a network with an MTU of 8896 bytes and set up firewall rules. (Creating a network requires `compute.networks.create` permission in your project)
+     ```
+     gcloud compute networks create mtu9k --mtu=8896 --project={PROJECT} --subnet-mode=auto --bgp-routing-mode=regional
```
Comment (Obliviour) [inline v5e_high_performance.md:49]: nit $ in front of {PROJECT} + other PROJECT variable on line 52 and incorrect $ position on line 58 for PROJECT

Comment (raymondzouu) [reply v5e_high_performance.md]: Good catch! Thank you

**Review by rwitten — APPROVED** [review]

**Pull request merged**